test/tools/api.js

```diff
@@ -145,3 +145,844 @@ module.exports.classNames = new Set(api.map(({ className }) => className))
 module.exports.classNameToMethodList = new Map(api.map((api, _, array) =>
   [api.className, sorted(Array.from(new Set(Array.from(array.filter(v => v.className === api.className), method => method))), (a, b) => byStrings(a.method, b.method))]
 ));
+
+/**
+ * Generated from `api`, this is an exhaustive list of all methods we can unit test.
```

Comment:
Generated how? would we need to regen it ever? (prob not but just wanna capture the history of how at least)

Reply:
works for me, and I agree, low traffic here